### ISSUE TITLE
Edited Live Photo export finishes with error

### ISSUE THREAD

**Issue by mave2k** [RhetTbull/osxphotos#576]
**Version**

v0.44.6 CLI ( osxphotos_MacOS_exe_darwin_x64_v0.44.6.zip)
used with standard Mac Terminal.app
downloaded from releases page

**Description**

When an edited live photo is exported with the CLI, the export finishes with an error

```
Exporting IMG_3449.HEIC (261E1200-8173-4152-961C-70A7A897BA4F.heic) as IMG_3449.HEIC
Exported /Users/USERNAME/Desktop/photos/gamma/IMG_3449.HEIC
Exported /Users/USERNAME/Desktop/photos/gamma/IMG_3449.mov
Exporting edited version of IMG_3449.HEIC (261E1200-8173-4152-961C-70A7A897BA4F.heic) as IMG_3449_edited.heic
Error exporting photo (261E1200-8173-4152-961C-70A7A897BA4F: IMG_3449.HEIC) as IMG_3449_edited.heic: local variable 'dest_edited' referenced before assignment
```

**To Reproduce**
1. What' the full command line you used with osxphotos?
2. What was the error output?

**Result**

The original photo `.HEIC` and its `.mov` video are exported correctly. Also, an `_edited.heic` is exported, despite the error showing. However, no video portion of the edited image is exported.
The error message does **not** abort further exporting.

Files Result Example:
```
IMG_3449.HEIC
IMG_3449.mov
IMG_3449_edited.heic
```

**Expected behavior**

According to the docs, I would have expected a `_edited.mov` file in addition to the three files generated and no error message here.

**Screenshots**
<img width="1153" alt="image" src="https://user-images.githubusercontent.com/8629837/148660874-c33c505f-fb4b-4d6d-9651-145a864b695f.png">

**Desktop (please complete the following information):**
 - OS: macOS Monterey v12.0.1 on an Intel i7 CPU
 - osxphotos version v0.44.6

**Additional context**
Tried first in a compound command with additional options (`--directory` and `folder`) activated. But also happens with a plain export without additional arguments.

**Comment by RhetTbull** [comment]
This was a bug introduced in some recent refactoring and it unfortunately escaped the otherwise robust test suite.  I've fixed it in [v0.44.7](https://github.com/RhetTbull/osxphotos/releases/tag/v0.44.7).  Let me know if you have any other issues!

**Comment by mave2k** [comment]
Works now for me with CLI 0.44.7:
Live Photos now correctly export to:
`IMG.HEIC`
`IMG.mov`
`IMG_edited.heic`
`IMG_edited.mov`
without any error message
